tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1789,6 +1789,12 @@ pfcwd/test_pfcwd_all_port_storm.py:
       - "hwsku in ['Arista-7060X6-64PE-256x200G']"
       - "topo_type in ['m0', 'mx', 'm1']"
 
+pfcwd/test_pfcwd_cli.py:
+   xfail:
+     reason: "Skip due to https://github.com/sonic-net/sonic-mgmt/issues/18496"
+     conditions:
```

Comment:
Thanks for your PR, I reproduced the issue locally and are working on it with Arista team. 
May you please let me know if the case failed on all hwskus or not? and if there is any blocker from your side? 
thanks.

Reply:
Hi @lipxu the issue reproduced on any hwsku, when there is more than one portchannell memeber in the portchannel, and the portchannel was selected to be used in the test case, it will fail.

Reply:
@vivekverma-arista  Could you please help to check the issue? If it's the issue and cannot be fixed in short time, Could we revert the original PR first? Thanks.

Reply:
> test

HI @vivekverma-arista Can you assign the github issue to yourself, and update the status once it is fixed?

Reply:
> @lipxu can you approve, then we can xfail it until the github issue is fixed.

Hi, @nhe-NV , may I know which test this failure is blocking, and on which branch? If it's a feature branch, can we mark the xail only for this feature branch, while keeping the failure visible on master, thanks.

Reply:
It is failing on the 202505, I will convert it to 202505 PR.